### DISCUSSION TITLE
Deploying to Google Cloud Platform (GCP)

### DISCUSSION THREAD

**Discussion by realB12** [umputun/remark42#1540] [Q&A]
Has anyone detailed instructions about **how to deploy the BackendServer to Google Cloud Platform (GCP)**? Has anyone done this already and wants to earn some money to assist me doing so? 
I am not even sure, what GCP service sould be taken: I am assuming `Compute Engine` will work whereas `Cloud Run`, or` App Engine` will not work because of BoltDB needs to be persistent (same issue as for Deploying in Heroku #1067). 
Any support or indication where to look up things is highly appreciated. 
Happy coding and kind regards
René

**Comment by paskal** [comment]
I believe [this remark42 installation instruction for the cloud](https://terrty.net/2022/self-hosted-comments-remark42/) is what you are looking for. It's not very detailed, but it outlines all the significant steps you have to take, so you should be able to make it work with enough patience.

**Reply by realB12** [reply]
I have read your document on Oracle and Digital Ocean setup. However, on GCP there are a lot more decisions to make and various operation and configuration models to choose from. Especially when it comes to security and remote debugging things are no longer trivial. 
So I really need a GCP-Pro on this. (Currently I am able to run the server on a Cmpute Engine - but I am really not sure, whether this is the best setup for a professional and scalable environment).

**Reply by paskal** [reply]
I'm sorry for not being clear on that. You need to use the GCP [Compute Engine](https://cloud.google.com/compute) with the cheapest machine available to start and then follow the generic setup instructions. However, you must learn how to SSH to the machine to follow the instructions, I see that as the most challenging part of the instruction.

**Reply by realB12** [reply]
yeah, this is the road what I am currently going. I will write a doc when successful. Thanks for your support.
